setup-machine.sh

```diff
@@ -23,7 +23,7 @@ MODEL_NAMES+=('VxAdmin')
 
 echo "${#CHOICES[@]}. Ballot Scanner"
 CHOICES+=('bsd')
-MODEL_NAMES+=('VxScan')
+MODEL_NAMES+=('VxCentralScan')
 
 echo "${#CHOICES[@]}. Ballot Marking Device (BMD)"
```

Comment:
wondering if it matters that the bmd always has model name set to VxMark here

Reply:
When we configure the machines there is a way to get into an admin menu (tty2, you press, f2 or f3 or something...) where Drew (or whoever) can change the app mode, which also changes the model name. You can see that in the admin_functions scripts directory if you are curious.

Reply:
Oh maybe I don't actually change the model name I thought I did lol, regardless this environment variable isn't used for anything yet so its not a big deal.